app/src/main/java/com/example/boeing301house/ItemViewActivity.java

```diff
@@ -268,6 +270,11 @@ public void onCancel() {
         exitAddEditFragment();
     }
 
+    @Override
+    public void onDateRangeSelected(Calendar start, Calendar end) {
```

Comment:
itemview shouldnt have ondaterangeselected

Reply:
I thought that too but it creates a error if i erase

Reply:
its bc u have it in the addeditlistener